docs/FAQ/gui-faq-es.md

```diff
@@ -79,7 +79,8 @@ Pero tal como muchas cosas que evolucionan como el DNS, la situación es
 cambiante en el tiempo, y lo que es un aviso hoy puede ser un error
 mañana. Si realmente crees que hemos cometido un error en nuestro
 juicio, por favor no dudes en enviarnos un email a 
-zonemaster-devel@lists.iis.se con un enlace a tu prueba y una explicación
+zonemaster-users@lists.iis.se (moderated mailing list) con un enlace a tu prueba
```

Comment:
```suggestion
zonemaster-users@lists.iis.se (lista de correo moderada) con un enlace a tu prueba
```

Reply:
Fixed.

Reply:
Need a space in "sende en e-post til "

- sende en e-posttil zonemaster-users@lists.iis.se (moderated mailing list) med en lenke
+ sende en e-post til zonemaster-users@lists.iis.se (moderert e-postliste) med en lenke